compatibility-verifier/sample-test-suite/config/query-results/feature-test-1-rest-sql.results

```diff
@@ -18,7 +18,7 @@
 #
 
 # Aggregation
-{"resultTable":{"dataSchema":{"columnDataTypes":["LONG"],"columnNames":["count(*)"]},"rows":[[10]]},"exceptions":[],"numServersQueried":1,"numServersResponded":1,"numSegmentsQueried":1,"numSegmentsProcessed":1,"numSegmentsMatched":1,"numConsumingSegmentsQueried":0,"numDocsScanned":10,"numEntriesScannedInFilter":0,"numEntriesScannedPostFilter":0,"numGroupsLimitReached":false,"totalDocs":10,"timeUsedMs":4,"segmentStatistics":[],"traceInfo":{},"minConsumingFreshnessTimeMs":0}
```

Comment:
this is incredibly misleading, 10 docs were not scanned, we want to do calculations based off these numbers and we need them to be correct. However, I imagine `MetadataBasedAggregationOperator` lied about this for a good reason?